sunpy/lightcurve/sources/noaa.py

```diff
@@ -44,30 +44,31 @@ class NOAAIndicesLightCurve(LightCurve):
     * `Indices Descriptions <ftp://ftp.swpc.noaa.gov/pub/weekly/README3>`_
     * `NOAA plots of Solar Cycle Progression <http://www.swpc.noaa.gov/products/solar-cycle-progression>`_
     * `NOAA Product List <http://www.swpc.noaa.gov/products-and-data>`_
+
+
     """
+    default_url_noaa_indices = "ftp://ftp.swpc.noaa.gov/pub/weekly/RecentIndices.txt"
 
     def peek(self, type='sunspot SWO', **plot_args):
         """Plots NOAA Indices as a function of time. An example is shown below.
 
-        .. plot::
```

Comment:
I think the [`plot::` directive]( http://matplotlib.org/devel/documenting_mpl.html#figures) is needed so the plot appears in [the documentation](http://sunpy.readthedocs.org/en/latest/api/sunpy.lightcurve.NOAAIndicesLightCurve.html#sunpy.lightcurve.NOAAIndicesLightCurve.peek).